src/models/Illness.ts

```diff
@@ -1,7 +1,10 @@
 
-import { YesNo } from './fields/YesNo';
+import { IllPerson } from 'app/models/fields/IllPerson';
+import { YesNo } from 'app/models/fields/YesNo';
```

Comment:
Why does we need YesNo here?

Reply:
It just makes mapping from the user input easier (than converting a yes/no input to a boolean.. aye, it's likely trivial, but if we already have an enum defined then why not use it?) and will presumably make it slightly easier when presenting the data back to the user.

Reply:
Ah i get it, okay thats fine :)